[DOCS-5328] Add beta docs for Oracle (#17828)
DataDog/documentation: content/en/database_monitoring/setup_oracle/_index.md

---
title: Setting up Oracle
kind: documentation
description: Setting up Database Monitoring on an Oracle database

> disable_sidebar: true

Reviewer: ```suggestion
disable_sidebar: true
private: true
is_beta: true
```

Reviewer: Adding the above front matter will hid the article from appearing in search results. This is standard for private betas, although let me know if you'd prefer that it be discoverable via search.

---

{{< site-region region="gov" >}}
<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
{{< /site-region >}}

<div class="alert alert-info">

> The features described on this page are in beta. Contact your Customer Success Manager to get the required agent build and installation directions.

Reviewer: ```suggestion
The features described on this page are in private beta. Contact your Customer Success Manager for information on the required Agent build and installation instructions.
```

</div>

### Oracle versions supported

|  | Self-hosted |
|--|------------|
| Oracle 19c | {{< X >}} |
| Oracle 21c | {{< X >}} |

The following deployment configurations are supported: on-prem, single-node, multi-tenant. Note these do not include RAC and Exadata or legacy architectures.

For setup instructions, select your hosting type:

> {{< partial name="dbm/dbm-setup-oracle" >}}

Author: yes, we will support many more deployment types in the future
